doc/sphinx/source/animations.rst

```diff
@@ -145,7 +142,7 @@ Here's the example from above as spritesheets:
          hotspot = { 5, 7 }
          sound_effect = {
             path = "sound/foo/bar",
-            priority = 128
+            priority = 50
```

Comment:
This might seem to be very picky but the text says this is the example from above so we need to have the allow_mulriple here as well to be precise

Reply:
Picky is good. Fixed.